tests/integration/test_vulnerability_detector/test_scan_results/data/test_cases/cases_duplicate_feeds.yaml

```diff
@@ -0,0 +1,61 @@
+- name: 'RedHat 8'
```

Comment:
```suggestion
- name: 'RedHat'
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800